### PR TITLE
Revert "Merge branch 'add/7596-transactions-filters' into develop"

### PR DESCRIPTION
**Pull request by mordeth**
#### Changes proposed in this Pull Request
This PR reverts accidentally merged commit 1db31c87c2e6c847d1a070bcf8aed61726c4a75d, reversing changes made to 48573837c3fd24027a5528761a3dd2cf0ba450e1.

#### Testing instructions
- Ensure the merge is correctly reverted

-------------------

- [x] Run `npm run changelog` to add a changelog file, choose `patch` to leave it empty if the change is not significant. You can add multiple changelog files in one PR by running this command a few times. 
- [x] Covered with tests (or have a good reason not to test in description ☝️)
- [x] Tested on mobile (or does not apply)

### CODE REVIEW

**Comment by botwoo** [comment]
### Test the build 

#### Option 1. Jetpack Beta

- Install and activate [Jetpack Beta](https://jetpack.com/download-jetpack-beta/).
- Use this build by searching for PR number `7745` or branch name `revert/7596-transactions-filter-branch` in your-test.site/wp-admin/admin.php?page=jetpack-beta&plugin=woocommerce-payments          

#### Option 2. Jurassic Ninja - available for logged-in A12s

:rocket: [Launch a JN site with this branch](https://jurassic.ninja/create/?jetpack-beta&shortlived&nojetpack&woocommerce&woocommerce-payments-dev-tools&branches.woocommerce-payments=revert/7596-transactions-filter-branch) :rocket:

:information_source: Install this [Tampermonkey script](https://github.com/Automattic/woocommerce-payments/tree/develop/bin/wcpay-live-branches) to get more options.

---

Build info: 

- Latest commit: ec10651e9faa467018bfbdcfed6d27b1f12eba5a
- Build time: 2023-11-20 14:52:28 UTC

_Note: the build is updated when a new commit is pushed to this PR._

File: changelog/revert-7596-transactions-filter-branch [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Revert 7596-transactions-filter
```
Comment (deepakpathania) [inline changelog/revert-7596-transactions-filter-branch]: Please prefix it with `Comment: `

**Review by deepakpathania — COMMENTED** [review]
Need to update changelog, rest looks ok.

**Review by dpaun1985 — APPROVED** [review]

File: changelog/revert-7596-transactions-filter-branch (hunk 1 above)
Comment (mordeth) [reply changelog/revert-7596-transactions-filter-branch]: Updated

**Pull request merged**